sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -243,7 +243,10 @@ def visit_transform(self, transform_node):
               pcoll.element_type, transform_node.full_label)
           key_type, value_type = pcoll.element_type.tuple_types
           if transform_node.outputs:
-            transform_node.outputs[None].element_type = typehints.KV[
+            key = (
+                None if None in transform_node.outputs.keys()
+                else next(iter(transform_node.outputs.keys())))
+            transform_node.outputs[key].element_type = typehints.KV[
```

Comment:
Prefer `only_element(transform_node.outputs.values())`

Reply:
Because GBK can only have one output (otherwise this could result in a failure) ? Used only_element() in few other locations where it's safe to use.

Reply:
Yes, so you can use only_element here as well.